src/Symfony/Component/Form/EnumFormTypeGuesser.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form;
+
+use Symfony\Component\Form\Extension\Core\Type\EnumType as FormEnumType;
```

Comment:
```suggestion
use Symfony\Component\Form\Extension\Core\Type\EnumType;
```

I don't see why we need the alias here

Reply:
Done. (Was previously needed because `EnumType` from `symfony/type-info` was also used.)